src/Tickets/Commerce/Reports/Orders.php

```diff
@@ -106,11 +106,11 @@ public function get_event_link( $event_id, $url_only = false ) {
 		 *
 		 * @since 5.2.0
 		 *
-		 * @var string $report_url Report URL
-		 * @var int    $event_id   The post ID
-		 * @var array  $ticket_ids An array of ticket IDs
-		 *
 		 * @return string
+		 * @var int    $event_id   The post ID.
+		 * @var array  $ticket_ids An array of ticket IDs.
+		 *
+		 * @var string $report_url Report URL.
```

Comment:
These should be above the `@return`, and they should be `@param` instead of `@var`.

Reply:
Oh, I didn't see that. Fixed.